proto/pocket/application/application.proto

```diff
@@ -5,12 +5,14 @@ option go_package = "pocket/x/application/types";
 
 import "cosmos_proto/cosmos.proto";
 import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
 import "pocket/shared/service.proto";
 
 // Application defines the type used to store an on-chain definition and state for an application
 message Application {
   string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application using cosmos' ScalarDescriptor to ensure deterministic encoding
   cosmos.base.v1beta1.Coin stake = 2; // The total amount of uPOKT the application has staked
   repeated shared.ApplicationServiceConfig service_configs = 3; // The ID of the service this session is servicing
+  repeated google.protobuf.Any delegatee_gateway_pub_keys = 4 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey", (gogoproto.nullable) = false]; // The corresponding cosmos.crypto.PubKey (interface) encoded into an cosmos.codec.Any for use in the non nullable slice of delegatee Gateways the application is delegated to.
```

Comment:
1. I was following this comment until the "for use ..." part. To me, it sounds like it's referring to some other place in code but it also sounds like it's describing this field, which I find a bit confusing; to say something like "this field holds X for use in this field" or "...where this field is used". Perhaps I'm misunderstanding but does that make sense?
2. #PUC on what the `[(cosmos_proto.accepts_interface)]` option is and/or where to learn more.